api/permissions_requests_handlers_test.go

```diff
@@ -0,0 +1,76 @@
+// +build integration
+
+package api_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+	"testing"
+
+	helpers "github.com/topfreegames/Will.IAM/testing"
+)
+
+func TestPermissionsRequestsListHandler(t *testing.T) {
+	helpers.CleanupPG(t)
+	saUC := helpers.GetServiceAccountsUseCase(t)
+	sa, err := saUC.CreateKeyPairType("some sa")
+	if err != nil {
+		t.Errorf("Unexpected error %s", err.Error())
+		return
+	}
+	app := helpers.GetApp(t)
+	req, _ := http.NewRequest("POST", "/permissions/requests", strings.NewReader(`{
+"service": "SomeService",
+"ownershipLevel": "RL",
+"action": "SomeAction",
+"resourceHierarchy": "*",
+"message": "hey, can I have this permission?"
+	}`))
+	req.Header.Set("Authorization", fmt.Sprintf(
+		"KeyPair %s:%s", sa.KeyID, sa.KeySecret,
+	))
+	rec := helpers.DoRequest(t, req, app.GetRouter())
+	if rec.Code != http.StatusCreated {
+		t.Errorf("Expected status 201. Got %d", rec.Code)
```

Comment:
t.Fatalf? If you got the wrong status, then everything else is probably failing too.